Framework/include/QualityControl/QualitiesToTRFCollectionConverter.h

```diff
@@ -0,0 +1,60 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   QualitiesToTRFCollectionConverter.h
+/// \author Piotr Konopka
+///
+
+#ifndef QUALITYCONTROL_QUALITIESTOTRFCOLLECTIONCONVERTER_H
+#define QUALITYCONTROL_QUALITIESTOTRFCOLLECTIONCONVERTER_H
+
+#include "QualityControl/QualityObject.h"
+#include <DataFormatsQualityControl/FlagReasons.h>
+#include <DataFormatsQualityControl/TimeRangeFlagCollection.h>
```

Comment:
I think but am not certain that you could forward declare it. Also, is the one above necessary ?

Reply:
you're right, i'll fix it.